nasa/zarr-eosdis-store
# Harmony 1756 - Use trusted publisher for publishing to PyPi

Changes GitHub publish action to use trusted publisher token instead of password stored in GitHub secret.

To test:

1. Click on the 'Releases' link in the main page of this GitHub repo
2. Click on 'Draft release' button.
3. Create a dummy tag
4. Choose this branch instead of 'main'
5. Fill in title and description with whatever
6. Choose "pre-production release" checkbox
7. Click on 'Publish release'
8. Check to the 'Actions' tab on the page and make sure the publish action succeeds
9. Go to pypi.org and search for harmony-service-lib
10. Check on the releases and verify that your release is there
11. Delete your test release
